tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"module": "commonjs",
+		"target": "es5",
+		"noImplicitAny": true,
+		"jsx": "react",
+		"outDir": "./dist/",
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitReturns": true,
+		"noImplicitThis": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"pretty": true,
+		"removeComments": true,
+		"strictNullChecks": true
+	},
+	"include": [
+		"./app/**/*"
+	],
+	"exclude": [
+		"node_modules"
+	]
```

Comment:
It says "Files included using "include" can be filtered using the "exclude" property. ". Also looks like "node_modules" is excluded by default. Does it work when you remove the "exclude" part?